pkg/utils/utils.go

```diff
@@ -181,12 +182,21 @@ func GetFileContents(filePath string) []byte {
 	return contents
 }
 
+const defaultShareDir = "/usr/share/logging"
+
 func GetShareDir() string {
-	shareDir := os.Getenv("LOGGING_SHARE_DIR")
-	if shareDir == "" {
-		return "/usr/share/logging"
+	// shareDir is <repo-root>/files - try to find from working dir.
+	const sep = string(os.PathSeparator)
+	const repoRoot = sep + "cluster-logging-operator" + sep
```

Comment:
When is this ever true?  I recall even the built image writes to something like `/go/src/openshift/....`

Reply:
Dunno, I was copying the old behaviour which uses this as the final fallback. If it's never used then we should perhaps make it an an error/panic if we get this far.

Reply:
You should use fmt.Sprintf instead string + string + string

Reply:
Can I have an LGTM?